bin/cli.js

```diff
@@ -70,6 +70,20 @@ For more information, see https://webpack.js.org/api/cli/.`);
 		try {
 			options = require("./utils/convert-argv")(argv);
 		} catch (err) {
+			if (err.code === "MODULE_NOT_FOUND") {
+				let errorMessage =
+					"\n\u001b[31mwebpack not found, \u001b[33mplease install webpack using\n\t\u001b[32mnpm install --save-dev webpack\n";
```

Comment:
`webpack not installed, consider installing it using npm install --save-dev webpack`

Reply:
Indentation seems weird here, you don't need different color schemes for the `not found` part, the `please install ` part and the installation text. Two colors is sufficient, one for the error stack and one for the npm save

Reply:
Done